src/_press/cal-itp-payments-data-dashboard.md

```diff
@@ -4,4 +4,5 @@ title: Cal-ITP’s free dashboard serves up ridership metrics for transit agenci
 external: https://gcn.com/data-analytics/2023/01/caltrans-serves-dashboard-metrics-local-transit-agencies/381473/
```

Comment:
@mrose914 Looks like GCN.com rebranded to Route Fifty. The gcn.com press link now permanently redirects to https://www.route-fifty.com/digital-government/2023/01/caltrans-serves-dashboard-metrics-local-transit-agencies/381473/. Should I rename the outlet and change the link to the new one?

Reply:
@mrose914 Oh oops. I was loking at the [Cal-ITP Media Coverage and Publications](https://docs.google.com/spreadsheets/d/12zdDDRArtyXmm1wZJk4-vq1KvF8aZ9BWrIBIM4T7R3w/edit#gid=1455215505).